keps/sig-storage/20180515-svcacct-token-volumes.md

```diff
@@ -159,6 +162,40 @@ sources:
     audience: ca.istio.io
 ```
 
+### File Permission
+
+
```

Comment:
State what the problem is. Namely that permissions changed to 0600 and how it breaks users.

Reply:
done.